stable/k8s-inventory/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: k8s-inventory
-version: 0.0.2
+version: 0.0.4
```

Comment:
@anchore/devops I've bumped to 0.0.4 so this chart is inline with anchore-charts-dev so the rebase is clean but if you want this to go back to 0.0.3 and we clean up anchore-charts-dev that's fine too just let me know